lib/awful/client.lua

```diff
@@ -986,17 +984,17 @@ end
 -- @return The property.
 -- @deprecated awful.client.property.get
 function client.property.get(c, prop)
-    if not client.data.persistent_properties_loaded[c] then
-        client.data.persistent_properties_loaded[c] = true
+    if not c.data._persistent_properties_loaded then
+        c.data._persistent_properties_loaded = true
         for p in pairs(client.data.persistent_properties_registered) do
             local value = c:get_xproperty("awful.client.property." .. p)
             if value ~= nil then
                 client.property.set(c, p, value)
             end
         end
     end
-    if client.data.properties[c] then
-        return client.data.properties[c][prop]
+    if c.data.awful_client_properties then
+        return c.data.awful_client_properties[prop]
```

Comment:
You might as well just return `c[prop]` and print a deprecated message now that this function is "really" deprecated. Until now, it was still used in the background by the miss handler.